test.js

```diff
@@ -1,22 +1,47 @@
-import test from 'ava';
-import fn from './';
+const test = require('ava');
+const tasklist = require('./');
```

Comment:
I didn't like the inconsistency of having require statements in the index file and import statements in the test file. But I guess your plan is to change to import statements everywhere in the long run, so yeah, gladly changing it back. :)